tools/olm/docker-entry.sh

```diff
@@ -102,6 +96,10 @@ generate_olm_bundle() {
   yq ea -i ".spec.replaces = \"${PACKAGE_NAME}.v${PREVIOUS_BUNDLE_VERSION}\" | .spec.replaces style=\"\"" "${CSV_FILE}"
 
   yq ea -i "del(.subjects[0].namespace)" "${ROLE_BINDING}"
+
+  # Needed to replace description with new bundle values
+  sed -i "s/release-1.1/release-${BUNDLE_VERSION}/" "${CSV_FILE}"
+  sed -i "s/version: 1.2.0/version: ${BUNDLE_VERSION}/" "${CSV_FILE}"
```

Comment:
would it make sense to add some nicer placeholder strings in the CSV_FILE ?

Reply:
That's a great idea @gyfora, I've added a RELEASE_VERSION placeholder and also I've added a few items under links, specifically:
    - name: Website
      url: https://flink.apache.org/
    - name: Documentation
      url: https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/
    - name: Mailing list
      url: https://lists.apache.org/list.html?dev@flink.apache.org
    - name: Slack
      url: https://apache-flink.slack.com/join/shared_invite/zt-1llkzbgyt-K2nNGGg88rfsDGLkT09Qzg#/shared-invite/email
    - name: GitHub
      url: https://github.com/apache/flink-kubernetes-operator

Reply:
@jbusche Links look good!
![image](https://user-images.githubusercontent.com/7155778/208775908-d5e02c9a-67f7-4e19-b487-59f127a725ce.png)